File: LpSolve/Result/SeidelResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using LpSolve.Elements;

namespace LpSolve.Result
{
    /// <summary>
    /// Base class for LP result
    /// </summary>
    public abstract class SeidelResult
    {
        public Point Point { get; protected set; }

        public abstract SeidelResult Resolve(HalfSpace halfSpace);
    }
}

File: cpp/Result/SeidelResult.cpp
#include "SeidelResult.h"

using namespace LpSolveCpp::Elements;
namespace LpSolveCpp
{
	namespace Result
	{

		Point *SeidelResult::getPoint() const
		{
			return privatePoint;
		}

		void SeidelResult::setPoint(Point *value)
		{
			privatePoint = value;
		}
	}
}
